SpatialGDK/Documentation/content/how-to-use-schema.md

```diff
@@ -60,4 +72,7 @@ Whenever you generate schema, the GDK checks the [SchemaDatabase]({{urlRoot}}/co
 
 This means that if you have a class that only exists on one user's machine (for example, a newly created class, or a class used for local testing) then these classes are automatically removed from the SchemaDatabase file whenever another user generates schema.
```

Comment:
Have suggested defining what the SchemaDatabase file is in the first comment above. Otherwise, it's not clear what this file is, you have to sort of work it out, I think. (Unless there is some text I missed - highly likely!)
Should the filename be in backticks? Not sure?!